modules/serverless-overriding-pdbs-serving.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/config-ha-services/ha-replicas-serving.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-serving_{context}"]
+= Overriding disruption budgets
+
+A Pod Disruption Budget (PDB) is a standard feature of Kubernetes APIs that helps limit the disruption to an application when its pods need to be rescheduled for maintenance reasons.
+
+.Procedure
+
+* You can override the default PDB for a specific resource by modifying the `minAvailable` configuration value in the `KnativeServing CR`.
+
+.Example PDB with a `minAvailable` seting of 70%
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+ name: knative-serving
+ namespace: knative-serving
+spec:
+ podDisruptionBudgets:
+ - name: activator-pdb
+   minAvailable: 70%
+----
+
+[NOTE]
+====
+If you disable high-availability, for example, by changing the `high-availability.replicas` value to 1, make sure you also update the corresponding podDisruptionBudget `minAvailable` value to 0. Otherwise, the pod disruption budget prevents automatic cluster or operator upgrades.
```

Comment:
Same feedback on this note as in the other module, just combining it all in one comment here:
```suggestion
If you disable high-availability, for example, by changing the `high-availability.replicas` value to `1`, make sure you also update the corresponding `podDisruptionBudget` `minAvailable` value to `0`. Otherwise, the pod disruption budget prevents automatic cluster or Operator updates.
```